app/js/modal.ts

```diff
@@ -18,6 +18,23 @@ function closeDialog() {
 }
 
 /**
+ * Opens viewResultsModal box
+ */
+function openViewResultsModal() {
+    document.querySelector<HTMLElement>('#viewResultsModal').style.display = 'block';
+    document.querySelector<HTMLElement>('.overlay').style.display = 'block';
+}
+
+/**
+ * Closes viewResultsModal box
+ */
+function closeViewResultsModal() {
+    document.querySelector<HTMLElement>('#viewResultsModal').style.display = 'none';
+    document.querySelector<HTMLElement>('.overlay').style.display = 'none';
+}
+
+/**
+ *
```

Comment:
These functions have quite semantic and descriptive names so the comments are not needed.

Reply:
The exact functions are `openDialog()`, `openViewResultsModal()`, `closeViewResultsModal()`

Reply:
Have removed comments from openViewResultsModal() and closeViewResultsModal() but left openDialog() as it was historic

Reply:
sure